src/docs/sdk/performance/time-to-initial-full-display.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: "Time to Initial Display/Time to Full Display"
+---
+
+We recommend implementing this feature for mobile and desktop SDKs.
+
+The current screen load transactions timings are not reliable to understand the time spent in the screen creation.
+This is because they wait for any span created in the meantime, including any file I/O or network request.
```

Comment:
`l`:  A bit cleaner
```suggestion
The current screen load transactions timings are unreliable in understanding the time spent in the screen creation.
This is because they wait for any span created, including any file I/O or network request.
```